docs-ref-mapping/reference-unified.yml

```diff
@@ -321,6 +321,11 @@
       landingPageType: Service
       children:
       - 'azure-mgmt-containerregistry*'
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.ContainerRegistry.Client
+      landingPageType: Service
```

Comment:
We're going to want to put your readme here, instead of `landingPageType: Service`. The CI will prevent a non-existent readme from being picked up, so we should probably start with that.